functions/scheduled/events.js

```diff
@@ -131,7 +131,9 @@ module.exports.parseDescription = async function (summary, description, channelI
         }
     }
 
-    parameters.extra = lines[5] === undefined ? "" : lines[5];
+    if (lines[5] !== undefined || lines[5] !== "") {
```

Comment:
This condition will always be true. Should it be `&&` instead of `||`?

Reply:
Should just be not undefined. Fixed.